packages/account/src/Components/poi/status/unsupported/detail-component.jsx

```diff
@@ -4,8 +4,8 @@ import { Loading, Icon, Text } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { WS } from '@deriv/shared';
 import { UploadComplete } from '../upload-complete/upload-complete';
-import PoiUnsupportedFailed from 'Components/poi-unsupported-failed';
-import uploadFile from 'Components/file-uploader-container/upload-file';
+import PoiUploadFailed from 'Components/poi/poi-upload-failed';
```

Comment:
just renamed for improving readability and possibility to reuse this component if it will be necessary